### PR TITLE
Use queue proxy

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

**Comment by Zapadlo** [comment]
Big chunk of work +1

File: healthchecks.go [hunk 1]
```diff
@@ -36,16 +46,68 @@ func (bridge BridgeApp) checkForwardable() error {
...
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	return checkIfTopicIsPresent(body, bridge.consumerConfig.Topic)
+
 }
+
+func checkIfTopicIsPresent(body []byte, searchedTopic string) error {
+	strBody := string(body[1:len(body) - 1])
+	strClearedBody := strings.Replace(strBody, "\"", "", -1)
```
Comment (Nesze) [inline healthchecks.go]: I would rather have parsed the JSON.

File: kafka-bridge.go [hunk 2]
```diff
@@ -37,179 +15,134 @@ import (
...
-}
-
 func main() {
 	initLoggers()
 	logger.info("Starting Kafka Bridge")
 	if len(os.Args) < 2 {
 		panic("Conf file path must be provided")
 	}
-	conf := os.Args[1]
+	confPath := os.Args[1]
```
Comment (Nesze) [inline kafka-bridge.go]: You could have tried to get rid of the whole config, and replace it with flags.

**Comment by Nesze** [comment]
+1

**Comment by tamas-molnar** [comment]
lgtm

**Comment by tosan88** [comment]
+1

File: healthchecks.go (hunk 1 above)
Comment (BerniVarga) [inline healthchecks.go]: I changed it so, it's nicer. :) Thanks.

File: kafka-bridge.go (hunk 2 above)
Comment (BerniVarga) [inline kafka-bridge.go]: Ok. I did it.

**Pull request merged**